setup.py

```diff
@@ -102,7 +102,7 @@ def get_long_description():
     return req_contents
 
 if __name__ == '__main__':
-    setup(name='avocado',
+    setup(name='avocado-framework',
```

Comment:
Yeah, that's one of the misconceptions about package names and the module names, which kept me from doing this before. They're actually very different things.